app/jsx/components/TabMetricsComp.jsx

```diff
@@ -83,7 +92,7 @@ class TabMetricsComp extends React.Component {
             </table>
           </div>
           <div className="c-tabcontent__divide2x-child">
-            <img className="o-imagecontent" src="/images/sample_data.png" alt="sample data"/>
+            <div className='altmetric-embed' data-badge-type='donut' data-badge-details='right' data-doi="10.1038/nature.2012.9872"></div>
```

Comment:
@amardesi Great question! Looks like the converter isn't converting the DOI yet... adding a story.